app/views/courses/show.html.erb

```diff
@@ -225,9 +227,12 @@
                                       <% end %>   
                                     </th>
                                   <%else%>
-                                    <th class="search_icon text-center" scope="row">                             
-                                      <a href="#" data-toggle="tooltip" title= "This document Preview is not available"> <i class=" fa fa-search" style="color:gray;"></i></a> 
-                                    </th>  
+                                    <th class="search_icon text-center" scope="row">          
+                                      <button class="tooltip_button" data-balloon-pos="left" data-balloon-length="medium" data-balloon-pos="up" aria-label="Word Documents cannot be Previewable">
```

Comment:
I like visibility that the button adds on hover but when not hovered it doesn't quite match the background color.